django/db/migrations/autodetector.py

```diff
@@ -1080,11 +1080,31 @@ def generate_removed_fields(self):
             self._generate_removed_field(app_label, model_name, field_name)
 
     def _generate_removed_field(self, app_label, model_name, field_name):
+        field = self.from_state.models[app_label, model_name].get_field(field_name)
+        # You can't just remove NOT NULL fields with no default or fields
+        # which don't allow empty strings as default because the reverse
+        # migration will need a default value to populate existing records in
+        # the database while adding the field back.
+        temp_default = models.NOT_PROVIDED
```

Comment:
Why is this variable called "temp_default" ? 🤔  Why can't it just be "default" as the kwarg is named? 🤔

Reply:
hmm ,I think "default" will be better.🤔